.github/mergify.yml

```diff
@@ -26,6 +26,8 @@ pull_request_rules:
           {% for c in commits %}
           (cherry picked from commit {{ c.sha }})
           {% endfor %}
+
+           Ref #{{number}}
```

Comment:
i still think `Refs` is better. could you reconsider this suggestion?

Reply:
changed